library.json

```diff
@@ -6,14 +6,15 @@
         {
             "name": "R. Mingis",
             "maintainer": true
-        }
+                }
     ],
     "repository": {
         "type": "git",
          "url": "https://github.com/rwmingis/InterruptButton"
     },
     "version": "1.0.0",
     "frameworks": "Arduino",
+    "platforms": ["espressif32"],
```

Comment:
I am using strict verification of libraries in platformio:

```
lib_compat_mode = strict
lib_ldf_mode = deep+
```

So this change will allow the library to be accepted where it should be.